src/server.ts

```diff
@@ -0,0 +1,65 @@
+import fastify from 'fastify';
+import path from 'path';
+import { loadActionPlugins } from './components/postgresConnect';
+import {
+  saveFiles,
+  getFilename,
+  createFilesFolder,
+  filesFolderName,
+} from './components/fileHandler';
+import getAppRootDir from './components/getAppRoot';
+
+// Bare-bones Fastify server
+
+const startServer = async () => {
+  await loadActionPlugins(); // Connects to Database and listens for Triggers
+
+  createFilesFolder();
+
+  const server = fastify();
+
+  server.register(require('fastify-static'), {
+    root: path.join(getAppRootDir(), filesFolderName),
+  });
+
+  server.register(require('fastify-multipart'));
+
+  // File download endpoint (get by Database ID)
+  server.get('/file', async function (request: any, reply: any) {
```

Comment:
Types for request and reply

Reply:
I'm not sure how to Type these. The actual request and reply object are enormous, complicated things, so wouldn't want to do it manually. Fastify is supposed to have a @types file and I've installed it. And when I specifically install @types packages for these plugins I get "fastify-multipart provides its own type definitions, so you do not need this installed." 

Do you know what I'm missing here?